modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -278,55 +278,46 @@ class CV_EXPORTS Platform
     Impl* p;
 };
 
-/*
-//! @brief Attaches OpenCL context to OpenCV
-//
-//! @note Note:
-//    OpenCV will check if available OpenCL platform has platformName name,
-//    then assign context to OpenCV and call clRetainContext function.
-//    The deviceID device will be used as target device and new command queue
-//    will be created.
-//
-// Params:
-//! @param platformName - name of OpenCL platform to attach,
-//!                       this string is used to check if platform is available
-//!                       to OpenCV at runtime
-//! @param platfromID   - ID of platform attached context was created for
-//! @param context      - OpenCL context to be attached to OpenCV
-//! @param deviceID     - ID of device, must be created from attached context
+/**
+@brief Attaches OpenCL context to OpenCV
+@note
+  OpenCV will check if available OpenCL platform has platformName name,
+  then assign context to OpenCV and call clRetainContext function.
+  The deviceID device will be used as target device and new command queue
+  will be created.
+@param platformName - name of OpenCL platform to attach,
+                     this string is used to check if platform is available
+                     to OpenCV at runtime
+@param platformID   - ID of platform attached context was created for
```

Comment:
It is better to remove `-` symbol and manual alignment in all parameter descriptions:
```.cpp
/**
...
@param platformName name of OpenCL platform to attach, this string is used to check if platform is available to OpenCV at runtime
@param platformID ID of platform attached context was created for
@param context OpenCL context to be attached to OpenCV
@param deviceID ID of device, must be created from attached context
...
*/
```